core/templates/pages/error-pages/error-404/error-404-page.component.ts

```diff
@@ -16,27 +16,46 @@
  * @fileoverview Component for the error 404 page.
  */
 
-import { Component, OnInit } from '@angular/core';
+import { Component, OnInit, OnDestroy } from '@angular/core';
 
 import { UrlInterpolationService } from
   'domain/utilities/url-interpolation.service';
 import { PageTitleService } from 'services/page-title.service';
+import { TranslateService } from '@ngx-translate/core';
+import { Subscription } from 'rxjs';
```

Comment:
Move this above together with the import on line 19. Ditto in other files.

Reply:
Done!